geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/stats/GeoMesaMetadataStats.scala

```diff
@@ -45,6 +46,16 @@ class GeoMesaMetadataStats(val ds: AccumuloDataStore, statsTable: String)
 
   private val metadata = new AccumuloBackedMetadata(ds.connector, statsTable, new StatsMetadataSerializer(ds))
 
+  private val compactionScheduled = new AtomicBoolean(false)
+  private val lastCompaction = new AtomicLong(0L)
+
+  executor.scheduleWithFixedDelay(new Runnable() {
+    override def run(): Unit = if (compactionScheduled.compareAndSet(true, false) &&
```

Comment:
Do we want to reverse the comparisons in this test?  Don't we want to check if the last compaction is at least an hour old before risking updating ```compactionScheduled```?  Or am I just misunderstanding this test?